statsd/statsd.go

```diff
@@ -285,8 +285,8 @@ func (c *Client) Set(name string, value string, tags []string, rate float64) err
 }
 
 // Timing sends timing information, it is an alias for TimeInMilliseconds
-func (c *Client) Timing(name string, value float64, tags []string, rate float64) error {
-	return c.TimeInMilliseconds(name, value, tags, rate)
+func (c *Client) Timing(name string, value time.Duration, tags []string, rate float64) error {
+	return c.TimeInMilliseconds(name, float64(value.Nanoseconds())/float64(time.Millisecond), tags, rate)
```

Comment:
I tend to use `value.Seconds()*1000` since `Seconds` already returns a `float64`. I think it looks a bit cleaner, but I have no idea if one way is faster or if it even makes a difference.